### DISCUSSION TITLE
Sorc no mana

### DISCUSSION THREAD

**Discussion by FakeForFearofFlame** [isid0re/SoloLeveling#67] [Q&A]
Sorc ends up running pits etc for gold but never finds/picks up gold faster than she uses it. Ends up using all her new found gold on half a belt of mana pots, slams em back after 2 mobs then back to hitting 1 monster at a time with her wand.

She put zero points in energy so she runs out very quickly. Running for a couple days now tried starting 4 new sorcs but all end up stuck in a1 nm burnin through mana and chickening to fallens cause she's tryin to smack em

Any ideas?

**Comment by isid0re** [comment]
I hadn't ran into this problem, but you are not the first to mention it to me. I had delayed teleport till she was level 28 because she would run out of mana too quickly. Only thing I can think of as a quick fix would be to alter the build file stats to add more mana and force a respec in nightmare. 

The bot doesnt respec normally in nightmare so it wont affect the script. 

I've been testing the scripts timing and performance across all classes for areas to improve so I will also look at revising the stats on the sorc and changing buffer allocation. So we dont have this problem. Thanks.
